kube-deploy/roles/kube-addons/tasks/addons-dashboard.yml

```diff
@@ -14,7 +14,18 @@
 # limitations under the License.
 #
 
+#For ARM support only
+- name: Checking for ARM support
+  shell: uname -a | grep -i "arm"
```

Comment:
`arm64` boards actually return back `aarch64` and `arm` boards return `arm.*`.   We'll need to do a conditional logic similar to https://github.com/kubernetes/kubernetes/blob/85d328f8da12f0f0e5b7e6dd2356e56cd584fc3e/hack/local-up-cluster.sh#L234

From Raspberry Pi (32-bit ARM, `arm`)
```
$ uname -a
Linux rpi2 4.1.19-v7+ #858 SMP Tue Mar 15 15:56:00 GMT 2016 armv7l armv7l armv7l GNU/Linux
```

From Odroid C2 (32-bit ARM, `arm64`)
```
$ uname -a
Linux ravi-arm51 3.14.79-103 #1 SMP PREEMPT Fri Jan 13 07:06:12 BRST 2017 aarch64 aarch64 aarch64 GNU/Linux
```